build.gradle.kts

```diff
@@ -10,7 +10,7 @@ repositories {
 }
 
 group = "org.hisp.dhis.rules"
-version = "3.0.0-SNAPSHOT"
+version = "3.0.1-SNAPSHOT"
```

Comment:
Version 3.0.0 hasn't been released in production. Shouldn't we keep it in 3.0.0 here?

Reply:
Fixed.
I think I changed at some point because I wasn't sure core was using the right version.
I put it back to 3.0.0-SNAPSHOT